src/components/text-editor/link-menu/editor-link-menu.tsx

```diff
@@ -17,7 +26,7 @@ import { LimelInputFieldCustomEvent } from '../../../components';
  */
 @Component({
     tag: 'limel-text-editor-link-menu',
-    shadow: true,
+    shadow: { delegatesFocus: true },
```

Comment:
if this change didn't work or wasn't helping for the rest of what you did, just remove it. Otherwise it'll be misleading for the reviewer who will be trying to connect the dots.

Reply:
⚡